doc/sphinx-guides/source/installation/config.rst

```diff
@@ -1038,6 +1038,20 @@ See also these related database settings below:
 - :ref:`:Authority`
 - :ref:`:Shoulder`
 
+.. _doi.baseurlstringnext
+
+doi.baseurlstringnext
++++++++++++++++++++++
+
+Dataverse uses multiple APIs from DataCite:
+
+- The DataCite MDS API is older, XML-based, and configured using :ref:`doi.baseurlstring`.
+- The DataCite REST API is newer, JSON-based, and configured using ``doi.baseurlstringnext``.
+
+In production, ``doi.baseurlstringnext`` should be set to ``https://api.datacite.org``
```

Comment:
Since this is specific to DataCite, it might be better to name it for that - doi.dataciteapibaseurl or something. (FWIW - I could swear that at some point I discussed adding a second option versus swapping mds for api in baseurlstring with someone and decided against having a second option, but I can't find code nor conversation...)